services/fxaccounts/FxAccounts.jsm

```diff
@@ -372,6 +391,16 @@ function copyObjectProperties(from, to, thisObj, keys) {
 class FxAccounts {
   constructor(mocks = null) {
     this._internal = new FxAccountsInternal();
+    if (RUST_BACKEND) {
+      // FxAccounts functionality with the Rust backend.
+      this._rustFxABridge = new FxAccountsRustBridge({
+        fxaServer: ROOT_URL,
+        // clientId: FX_OAUTH_CLIENT_ID,
+        clientId: 'a2270f727f45f648',
```

Comment:
this is using the fenix ID for now because the desktop client is not configured properly

Reply:
filed https://github.com/mozilla-services/cloudops-deployment/pull/3829/files to fix this